release-team/role-handbooks/docs/Release-Timeline.md

```diff
@@ -732,17 +736,21 @@ Once release management team has successfully cut the release, Docs Lead will me
 the `Create a merge commit` method of merging.
 Do not delete the dev-[future-release] when GitHub asks.
 
+- Verify that the pull request has an `approved` and an `lgtm` label.
 - Remove the hold from the on-hold integration PR when needed and merge into `main`.
 - Check the [Netlify build logs](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys) to make sure the
 site builds successfully.
 - Once the site is up, validate the docs by checking the navigation, version dropdown, [generated APIs](https://kubernetes.io/docs/reference/),
 [documentation version](https://kubernetes.io/docs/home/supported-doc-versions/), and random clicks.
 > Note: Make a note of the commit hash of the integration branch merge
 
+> Note: In 1.28 the Docs Release Team had failing CLA checks while merging the integration branch, in hwich case they made a decision to not change history at all. Rather they merged the integration branch even with failing CLA checks since the commit did pass CLA when it was merged to `dev-1.28` and the docs team did not want to fix this by altering history due to the side effects of doing so.
```

Comment:
```suggestion
> Note: In 1.28 the Docs Release Team had failing CLA checks while merging the integration branch, in which case they made a decision to not change history at all. Rather, they merged the integration branch even with failing CLA checks since the commit did pass CLA when it was merged to `dev-1.28`, and the docs team did not want to fix this by altering history due to the side effects of doing so.
```

Reply:
Thanks!